apps/projects/app/components/Panel/NewProject/RepoSelector.js

```diff
@@ -83,50 +89,54 @@ class Repo extends React.Component {
 
   unselect = () => this.setState({})
 
+  repoList = (visibleRepos, repoArray) => visibleRepos.length > 0 ? (
+    <RadioList
+      items={repoArray}
+      selected={this.state.repoSelected}
+      onChange={this.onRepoSelected}
+    />
+  ) : (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        marginTop: '19px',
+        flexDirection: 'column',
+      }}
+    >
+      <Text.Block>
+        There are no repositories matching{' '}
+        <Text weight="bold">{this.state.filter}</Text>
+      </Text.Block>
+      <ClearSearch onClick={this.handleClearSearch}>
+        Clear Search
+      </ClearSearch>
+    </div>
+  )
+
+  stillLoading = () => !(this.props.data && this.props.data.viewer && this.props.data.viewer.repositories.edges.length > 0)
```

Comment:
[Apollo handles loading state on its own](https://www.apollographql.com/docs/react/essentials/queries.html#loading), so we should not use custom loading conditions at all

Reply:
Done.